### PR TITLE
Update / create keyboard support topics

### PR DESCRIPTION
**Pull request by czernous-devexpress**

### CODE REVIEW

File: concepts/05 UI Components/Autocomplete/15 Keyboard Support.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <td>Page Up / Page Down</td>
+        <td>Moves focus to the first/last list item.</td>
+    </tr>
+    <tr>
+        <td>Enter</td>
+        <td>Selects focused list item.</td>
+    </tr>
+    <tr>
+        <td>Esc</td>
+        <td>Close the dropdown menu.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Autocomplete/15 Keyboard Support.md]:
```suggestion
        <td>Closes the drop-down menu.</td>
```

File: concepts/05 UI Components/Autocomplete/15 Keyboard Support.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>&uarr; / &darr;</td>
+        <td>Moves focus to the previous/following list item.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Autocomplete/15 Keyboard Support.md]: Do you mean "menu item"?

File: concepts/05 UI Components/Calendar/20 Keyboard Support.md [hunk 3]
```diff
@@ -21,6 +21,22 @@ An end user can use the following keys to interact with the UI component.
         <td>Enter</td>
         <td>Selects the focused day.</td>
     </tr>
+    <tr>
+        <td>Ctrl + &uarr;</td>
+        <td>Increases zoom level.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/20 Keyboard Support.md]:
```suggestion
        <td>Increases <a href="/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#zoomLevel">zoomLevel</a>.</td>
```

File: concepts/05 UI Components/ColorBox/20 Keyboard Support.md [hunk 4]
```diff
@@ -29,6 +29,14 @@ An end user can use the following keys to interact with the UI component.
         <td>Enter</td>
         <td>Selects the color.</td>
     </tr>
+    <tr>
+        <td>Alt + &uarr;</td>
+        <td>Opens the popup.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/ColorBox/20 Keyboard Support.md]: We call it a "drop-down editor" in the docs.

```suggestion
        <td>Opens the drop-down editor.</td>
```

File: concepts/05 UI Components/ColorBox/20 Keyboard Support.md [hunk 5]
```diff
@@ -29,6 +29,14 @@ An end user can use the following keys to interact with the UI component.
...
+    </tr>
+    <tr>
+        <td>Alt + &darr; or Esc</td>
+        <td>Closes the popup.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/ColorBox/20 Keyboard Support.md]:
```suggestion
        <td>Closes the drop-down editor.</td>
```

File: concepts/05 UI Components/DateBox/20 Keyboard Support.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ An end user can use the following keys to interact with the UI component.
         <td>Closes the date/time picker.</td>
     </tr>
     <tr>
-        <td rowspan="4">Calendar</td>
+        <td rowspan="5">Calendar</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DateBox/20 Keyboard Support.md]:
```suggestion
        <td rowspan="6">Calendar</td>
```

File: concepts/05 UI Components/DateBox/20 Keyboard Support.md [hunk 7]
```diff
@@ -33,6 +33,14 @@ An end user can use the following keys to interact with the UI component.
         <td>Enter</td>
         <td>Selects the focused day.</td>
     </tr>
+    <tr>
+        <td>Alt + &uarr; / Alt + &darr;</td>
+        <td>Opens/closes the popup.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DateBox/20 Keyboard Support.md]:
```suggestion
        <td>Opens/closes the drop-down editor.</td>
```

File: concepts/05 UI Components/DateBox/20 Keyboard Support.md [hunk 8]
```diff
@@ -33,6 +33,14 @@ An end user can use the following keys to interact with the UI component.
...
+    </tr>
+    <tr>
+        <td>Esc</td>
+        <td>Closes the popup.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DateBox/20 Keyboard Support.md]:
```suggestion
        <td>Closes the drop-down editor.</td>
```

File: concepts/05 UI Components/DropDownBox/26 Keyboard Support.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>Alt + &darr;</td>
+        <td>Opens the popup.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownBox/26 Keyboard Support.md]:
```suggestion
        <td>Opens the drop-down field.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>&larr;</td>
+        <td>Moves focus to the action button when split button is enabled.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the action button when <a href="/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#splitButton">splitButton</a> is enabled.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>&rarr;</td>
+        <td>Moves focus to the toggle button when split button is enabled.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the toggle button when <b>splitButton</b> is enabled.</td>
```

File: concepts/05 UI Components/Calendar/20 Keyboard Support.md [hunk 12]
```diff
@@ -21,6 +21,22 @@ An end user can use the following keys to interact with the UI component.
...
+    </tr>
+    <tr>
+        <td>Ctrl + &darr;</td>
+        <td>Decreases zoom level.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/20 Keyboard Support.md]:
```suggestion
        <td>Decreases <b>zoomLevel</b>.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td rowspan="3">Enter / Space</td>
+        <td>Opens the popup when the toggle button is focused.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Opens the drop-down menu when the toggle button is focused.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>Simulates a click action when the action button is focused.</td>
+    </tr>
+    <tr>
+        <td>Selects list item when focused.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Selects a menu item when focused.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>&uarr; / &darr;</td>
+        <td>Moves focus to the previous/following item.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the previous/following menu item.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>&darr;</td>
+        <td>Opens the dropdown menu.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Opens the drop-down menu.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>Esc</td>
+        <td>Closes the dropdown menu.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Closes the drop-down menu.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>Page Up / Page Down</td>
+        <td>Moves focus to the first/last item of the list page.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the first/last menu item on the page.</td>
```

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>Home / End</td>
+        <td>Moves focus to the first/last list item.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the first/last menu item.</td>
```

File: concepts/05 UI Components/Lookup/30 Keyboard Support.md [hunk 20]
```diff
@@ -25,6 +25,14 @@ An end user can use the following keys to interact with the UI component.
         <td>Tab / Shift + Tab</td>
         <td>If the drop-down menu is opened, moves focus to the list / search field.</td>
     </tr>
+    <tr>
+        <td>Page Up / Page Down</td>
+        <td>Moves focus to the first/last item of the list's page.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Lookup/30 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the first/last menu item on the page.</td>
```

File: concepts/05 UI Components/Lookup/30 Keyboard Support.md [hunk 21]
```diff
@@ -25,6 +25,14 @@ An end user can use the following keys to interact with the UI component.
...
+    </tr>
+    <tr>
+        <td>Home / End</td>
+        <td>Moves focus to the list's first/last item.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Lookup/30 Keyboard Support.md]:
```suggestion
        <td>Moves focus to the first/last menu item.</td>
```

File: concepts/05 UI Components/Popup/20 Keyboard Support.md [hunk 22]
```diff
@@ -0,0 +1,25 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>&uarr; &rarr; &darr; &larr;</td>
+        <td>Drags the popup when it is focused and drag is enabled.</td>
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Popup/20 Keyboard Support.md]:
```suggestion
        <td>Drags the popup when it is focused and <a href="/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#dragEnabled">dragEnabled</a> is <b>true</b>.</td>
```

**Review by RomanTsukanov — APPROVED** [review]

File: concepts/05 UI Components/Autocomplete/15 Keyboard Support.md [hunk 23]
```diff
@@ -0,0 +1,33 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>&uarr; / &darr;</td>
+        <td>Moves focus to the previous/following menu item.</td>
```
[the remark this answers is not in the document]
Comment (ksercs) [reply concepts/05 UI Components/Autocomplete/15 Keyboard Support.md:10]: are u sure that `menu` is a good name?

File: concepts/05 UI Components/DropDownBox/26 Keyboard Support.md [hunk 24]
```diff
@@ -0,0 +1,21 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>Alt + &darr;</td>
+        <td>Opens the drop-down field.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/DropDownBox/26 Keyboard Support.md:10]: I'm not sure that `field` is a good name

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 25]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td rowspan="3">Enter / Space</td>
+        <td>Opens the drop-down menu when the toggle button is focused.</td>
+    </tr>
+    <tr>
+        <td>Simulates a click action when the action button is focused.</td>
+    </tr>
+    <tr>
+        <td>Selects a menu item when focused.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md:24]: `when focused` : is it clear what is focused?

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 26]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>&uarr; / &darr;</td>
+        <td>Moves focus to the previous/following menu item.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md:28]: when popup is opened

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 27]
```diff
@@ -0,0 +1,55 @@
...
+        <td>&darr;</td>
+        <td>Opens the drop-down menu.</td>
+    </tr>
+    <tr>
+        <td>Esc</td>
+        <td>Closes the drop-down menu.</td>
+    </tr>
+    <tr>
+        <td>Page Up / Page Down</td>
+        <td>Moves focus to the first/last menu item on the page.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md:40]: when popup is opened

File: concepts/05 UI Components/DropDownButton/90 Keyboard Support.md [hunk 28]
```diff
@@ -0,0 +1,55 @@
...
+    </tr>
+    <tr>
+        <td>Home / End</td>
+        <td>Moves focus to the first/last menu item.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/DropDownButton/90 Keyboard Support.md:44]: when popup is opened

File: concepts/05 UI Components/Calendar/20 Keyboard Support.md [hunk 29]
```diff
@@ -21,6 +21,22 @@ An end user can use the following keys to interact with the UI component.
...
+        <td>Ctrl + &uarr;</td>
+        <td>Increases <a href="/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#zoomLevel">zoomLevel</a>.</td>
+    </tr>
+    <tr>
+        <td>Ctrl + &darr;</td>
+        <td>Decreases <b>zoomLevel</b>.</td>
+    </tr>
+    <tr>
+        <td>Home</td>
+        <td>Moves focus to the first date in the view.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/Calendar/20 Keyboard Support.md:34]: when popup is opened? (and the same below)

File: concepts/05 UI Components/ColorBox/20 Keyboard Support.md [hunk 30]
```diff
@@ -29,6 +29,14 @@ An end user can use the following keys to interact with the UI component.
         <td>Enter</td>
         <td>Selects the color.</td>
     </tr>
+    <tr>
+        <td>Alt + &uarr;</td>
+        <td>Opens the drop-down editor.</td>
```
Comment (ksercs) [inline concepts/05 UI Components/ColorBox/20 Keyboard Support.md:34]: mb `opens the drop-down menu / popup`? (and the same below)

**Review by ksercs — APPROVED** [review]

**Pull request merged**